people/fani-maksakuli.md

```diff
@@ -0,0 +1,5 @@
+/Users/Fani/Downloads/Fani\ Maksakuli.jpg 
```

Comment:
This url will not work

See other PR's for an example of how to do this

Also syntax below will not render - see Google for Markdown syntax